### DISCUSSION TITLE
Formatter configuration

### DISCUSSION THREAD

**Discussion by zabil** [LunarVim/LunarVim#1940] [General]
Hi, LunarVim fan here.

I recently upgraded lunar vim  to `v0.6.1` and started seeing a deprecation notice on launch

```
09:25:25 [WARN ] lvim: "Deprecation notice: [lvim.lang.python] setting is no longer supported. See https://github.com/LunarVim/LunarVim#breaking-changes" file="init.lua", line=60
```

I tracked down the change to this configuration line in `config.lua`

```
lvim.lang.python.formatters = {
  {
    exe = "black",
  }
}
```

It looks like the new configuration from ` ~/.local/share/lunarvim/lvim/utils/installer/config.example.lua` is

```
local formatters = require "lvim.lsp.null-ls.formatters"
formatters.setup {
   { exe = "black", filetypes = { "python" } },
}
```

Changing it  to the new format solves the problem. Starting this discussion for reference in case anyone runs into the same issue. Happy to change any documentation related to this if https://www.lunarvim.org/languages/python.html is the right place to do that.

**Comment by peidrao** [comment]
Thanks man!
